### ISSUE TITLE
Explain in plain English the various secure DNS server choices

### ISSUE THREAD

**Issue by JonnyRedHed** [AdguardTeam/AdguardForAndroid#2553]
It would be good if you wrote explanations of each secure DNS server option ion the long list.    Maybe a pop up bubble when hovering over each DNS server.  Its hard to see the full name, or know much about it.  

*Edit, although just moving to this v3 beta its a lot better now.  Good job!

**Comment by ameshkov** [comment]
What kind of explanation you'd like to see? If you tap on a DNS provider, there is a list of its features, is it not enough?

**Comment by JonnyRedHed** [comment]
I was using the stable 2.xx until yesterday, now though on the 3.xx beta and your correct everything is much better to understand now. Good job.  Please close this issue.
